code/game/mecha/equipment/tools/mining_tools.dm

```diff
@@ -49,7 +49,7 @@
 		drill.occupant_message("<span class='danger'>[src] is too durable to drill through.</span>")
 
 /turf/closed/mineral/drill_act(obj/item/mecha_parts/mecha_equipment/drill/drill)
-	for(var/turf/closed/mineral/M in range(drill.chassis,1))
+	for(var/turf/closed/mineral/M in range(drill.chassis,3))
```

Comment:
this'll create a weird cone and the speed buff might be more than enough

Reply:
i think it'll be fine

Reply:
i suggest getting a picture.